test/plugin/scenarios/apm-toolkit-trace-scenario/src/main/java/test/org/apache/skywalking/apm/testcase/toolkit/controller/TestController.java

```diff
@@ -62,6 +62,14 @@ public String toolKitCase() {
                 // ignore
             }
         });
+        testService.asyncSupplier(()->{
+        	try {
+				visit("http://localhost:8080/apm-toolkit-trace-scenario/case/asyncVisit/runnable");
+			} catch (IOException e) {
```

Comment:
It should be ```http://localhost:8080/apm-toolkit-trace-scenario/case/asyncVisit/supplier ```

Reply:
@sxzaihua You should add your own servlet method(`supplier`), and follow the plugin test doc, to add your new segment to verify the plugin feature.